README.md

```diff
@@ -34,6 +34,7 @@ for a Python package.
     [PDM](https://pdm.fming.dev/) or
     [Hatch](https://hatch.pypa.io)
     [Maturin](https://pypi.org/project/maturin/0.8.2/)
+    [scikit-hep](https://scikit-hep.org)
```

Comment:
shouldn't it be scikit-build?

```suggestion
    [scikit-build](https://scikit-build.readthedocs.io/en/latest/)
```

Reply:
actually I copied the link from `Scientific Python`, they have provided the they used skbuild for scikit.

Reply:
yep skbuild is the name used to import scikit-build ... by the way, the scicookie from scientific python group is under `BSD 3-Clause License` if you are copying any code from there you need to ensure you are giving the credits (citation) properly

Reply:
maybe you can find some information here: https://opensource.stackexchange.com/questions/10685/can-someone-re-license-my-bsd-3-licensed-project-under-the-mit-license-remove-m

Reply:
but in general, I would say that would shouldn't copy any code from there .. just use it as reference